<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCreateUser</class>
 <widget class="QDialog" name="DialogCreateUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupInformationUser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Information</string>
   </property>
   <widget class="QLabel" name="labelEmailWarning">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>195</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>22</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblUserName">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPassWord">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblFullName">
         <property name="text">
          <string>Full name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblEmail">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPhone">
         <property name="text">
          <string>Phone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRole">
         <property name="text">
          <string>Role</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineUsUserName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineUsPassword"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineUsFullName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineUsEmail"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineUsPhone"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboRole"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
